package.json

```diff
@@ -36,8 +36,9 @@
     "@babel/core": "^7.26.0",
     "@babel/eslint-parser": "^7.25.9",
     "@babel/plugin-proposal-decorators": "^7.25.9",
-    "@ember/test-helpers": "^2.6.0",
     "@ember/optional-features": "^2.2.0",
+    "@ember/string": "^4.0.0",
+    "@ember/test-helpers": "^4.0.4",
```

Comment:
Can be upgraded now because the package.json is using ember 4+.
The ember-try scenario for 3.28 explicitly downgrades this again. The v4 of this dep doesn't support Ember 3.x.